src/main/java/org/jabref/gui/slr/ManageStudyDefinition.css

```diff
@@ -1,3 +1,6 @@
 .slr-tab {
     -fx-padding: 1em;
 }
+.warning-label {
+    -fx-text-fill: red;
+}
```

Comment:
Can you check whether there some more global definitions? I think, that should go into a more global .css file (so that it can be styled - see https://themes.jabref.org/)

Reply:
Will take a look.

Reply:
There is `Base.css`. It contains `.warning-message`. 

Proposal: use that https://github.com/JabRef/jabref/blob/9116477440218d4c0aafdf95fa64cbbbf379fa63/src/main/java/org/jabref/gui/Base.css#L1306